e2e/testinfra/terraform/common/source_repositories.tf

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_sourcerepo_repository" "configsync-ci-cc" {
+  name = "configsync-ci-cc"
+}
+
+resource "google_sourcerepo_repository" "csr-auth-test" {
```

Comment:
This repo was created from source in the `e2e/testdata/hydration/kustomize-components` directory, so I renamed the repo to `kustomize-components`: https://github.com/GoogleContainerTools/kpt-config-sync/blob/dd665512df93cff7860ebf6dc60154b9bcf4eae8/scripts/push-to-test-csr-repos.sh#L20.

Reply:
Done. PTAL